clouseau/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+import libmozdata.config as config
+
+
+here = os.path.abspath(__file__)
+path = os.path.join(here, '../config.ini')
```

Comment:
Not a problem directly, but I'm always worried when using the path separators directly for compatibility between platforms.